modules/exploits/unix/webapp/wp_slideshowgallery_upload.rb

```diff
@@ -0,0 +1,117 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::HTTP::Wordpress
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(update_info(
+      info,
+      'Name'           => 'Wordpress SlideShow Gallery Authenticated File Upload',
+      'Description'    => %q{
+          The Wordpress SlideShow Gallery plugin contains an authenticated file upload
+          vulnerability. We can upload arbitrary files to the upload folder, because
+          the plugin also uses it's own file upload mechanism instead of the wordpress
+          api it's possible to upload any file type. The user provided does not need
+          special rights, and users with "Contributor" role can be abused.
```

Comment:
@FireFart you're right, as a contributor is presenting a HTTP 500 error for me too.